### ISSUE TITLE
healthcare=pharmacy

### ISSUE THREAD

**Issue by UKChris-osm** [osmlab/name-suggestion-index#4853]
Having just added [Allied Pharmacies](https://github.com/osmlab/name-suggestion-index/commit/e6fac38940137f5b43a2df8993c5db1fa265b6f8), I noticed the two tags with the value "_pharmacy_".

I understand a while back the "_healthcare=*_" tag was introduced, and part of that was "_healthcare=pharmacy_" which is currently included as a tag for any pharmacy within the NSI. #57

However, I've just noticed on the [Healthcare Wiki](https://wiki.openstreetmap.org/wiki/Key:healthcare) that "_healthcare=pharmacy_" is crossed out as having been rejected as a tag, so should this be removed from the NSI, leaving just "_amenity=pharmacy_" as the primary tag, or is it still there for compatibility reasons?

**Comment by bhousel** [comment]
Yeah I did see that.. That edit on the wiki was made by one guy who doesn't like any of the healthcare tags, but they singled out the one tag that wasn't listed in the proposal specifically.  I'm not sure what to do about it.

At the time, we tried to support all the tags that the healthcare proposal included.  This included **dual-tagging** (not deprecating) the existing amenity tags: `amenity=hospital`. `clinic`, `doctors`, `dentist`.   I think most people would agree that `pharmacy` belongs in this list too, even though it was left out of the proposal.  (It was left out because the original proposal wanted to replace it with `shop=*`, which nobody wanted).

Again, the proposal originally wanted to _deprecate_ all these tags, instead the iD team took the middle ground of just adding a `healthcare` tag alongside the existing established `amenity` tag.  This approach doesn't break anything, but gives consumers the option to use the new `healthcare` tags if they want to.

I'm happy to discuss this rationally with people, but we can't do that if people just get mad and change the wiki after a decision has been made, or spin it like "iD invented a tag".  People asked for `healthcare=*` tags and we tried to do our best to add them.

https://github.com/openstreetmap/iD/pull/4329
https://github.com/openstreetmap/iD/issues/7300

I know lately @ZeLonewolf has been doing a lot of work to understand these old tagging inconsistencies, and has been really good at hearing the stories of how things ended up the way they did, and pushing back on some of the rhetoric that people post to the tagging list or the wiki.  Maybe this is another tag he'd be interested in cleaning up with a proper proposal?
